<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ku</class>
 <widget class="QDialog" name="ku">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextBrowser" name="ku_lei">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="ku_zhucitiao">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="ku_fuw1">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="ku_fun1">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>90</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="ku_fuw2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="ku_fun2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>150</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="ku_fuw3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="ku_fun3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="ku_fuw4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="ku_fun4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>270</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="ku_zhucitiao_number">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="xuanze">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="show">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>显示</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
